### DISCUSSION TITLE
Is it possible to have different sidebar for a page while using global sidebar for the rest?

### DISCUSSION THREAD

**Discussion by viksrc** [quarto-dev/quarto-cli#10506] [Q&A]
### Description

I am using level 1  # {.sidebar} to define a common sidebar with inputs across multiple pages in a dashboard.  I would like to define a different sidebar  for a specific page.  I tried defining level 2 ##{.sidebar} for the specific page however this ends up appending  inputs to those defined in the level 1 sidebar. 
Is this expected behavior?  Appreciate any suggestions/examples  on how to separate sidebar for a single page while at the same time using a global sidebar for the rest?

**Comment by mcanouil** [comment]
Yes, see the documentation: https://quarto.org/docs/websites/website-navigation.html#side-navigation
